client/src/components/DisplayInsurances.js

```diff
@@ -0,0 +1,155 @@
+import { useState } from "react";
+import styled from "styled-components";
+
+import { ReactComponent as reset } from "../images/reset.svg";
```

Comment:
```suggestion
import { ReactComponent as Reset } from "../images/reset.svg";
```
React Components should always start with capital letters